_config.yml

```diff
@@ -0,0 +1,30 @@
+title: Meteor Demo
+author: AIOps team
+# logo: logo.png
+execute:
+  execute_notebooks: false
+
+# Only build files specified by the _toc
+only_build_toc_files: true
+
+# Exclude any additional file types being built
+exclude_patterns: [.github/*, manifests/*,  CHANGELOG.md, docs/images/*]
+
+# Information about where the book exists on the web
+repository:
+  url: https://github.com/aicoe-aiops/meteor  # Online location of your book
+  path_to_book: ""  # Optional path to your book, relative to the repository root
+  branch: master  # Which branch of the repository should be used when creating links (optional)
+
+# Add GitHub buttons to your book
+# See https://jupyterbook.org/customize/config.html#add-a-link-to-your-repository
+html:
+  use_issues_button: true
+  use_repository_button: true
+  google_analytics_id: ""  # A GA id that can be used to track book views.
+
+# Launch button settings
+launch_buttons:
```

Comment:
not sure if the deployed version is old but it looks like there is still a "Binder" deploy button. It should be removed if not already.